sympy/core/relational.py

```diff
@@ -180,13 +203,25 @@ def canonical(self):
         """
         args = self.args
         r = self
-        if r.rhs.is_Number:
-            if r.lhs.is_Number and r.lhs > r.rhs:
+        if len(r.rhs.free_symbols) == 0:
```

Comment:
-> `if r.rhs.is_number`

Reply:
OK! So that is the same thing? I never really thought of digging into the `is_number` details, but that clearly gives more readable code.

Reply:
As was mentioned in another PR, there is a slight difference: `is_number` is something that can be evaluated to a number whereas something without free symbols might still have AppliedUndef present that would prohibit evaluation. So using `if not r.rhs.free_symbols` would allow `f(1)` to appear on the rhs whereas `if r.rhs.is_number` would only allow something like `sin(1)` -- things that can be evaluated.